erpnext/manufacturing/doctype/bom_operation/bom_operation.json

```diff
@@ -117,13 +118,21 @@
    "fieldname": "sequence_id",
    "fieldtype": "Int",
    "label": "Sequence ID"
+  },
+  {
+   "default": "0",
+   "description": "If checked, on Work Order time will be fixed (do not depends on quantity to produce)",
+   "fieldname": "fixed_time",
+   "fieldtype": "Check",
+   "in_list_view": 1,
```

Comment:
Have you considered alternatives like batch sizes and "BOM Qty"? That would solve the problem in most cases.


Also this behaviour has to be changed in other places where BOM operations table is used... like the BOM doctype.

Reply:
The batch size cannot be use is this use case, to make 
BOM for 1 cherry cake, I need article (flour, cherry, butter,etc...) and the first operation "warm'up the oven" (at 150 C°)  it take 15 mins. 
On the OF I can produce 1 or 150 cherry cake this time (and cost) will not change.
I don't want to make BOM for 1 cherry cake , BOM for 2 cherry cake, BOM for 3 cherry cake, etc...

On anoter BOM for blueberry cake, I need article (flou, blue berry, butter,etc...) and the first operation will be "warm'up the oven," (at 180 °C) it take 18 mins. 
On the OF I can produce 1 or 150 blueberry cake this time (and cost) will not change.
Same I don't want to make BOM for 1 blueberry cake, bom for 2 blue berry cake ,etc..

That why there is no other doctype to update, only the table child Doctype  bom_operation use into BOM 
![image](https://user-images.githubusercontent.com/1050053/130802577-147e8759-c095-4ea2-8033-f8d98c0f5bf5.png)